src/sql/workbench/contrib/notebook/browser/cellViews/textCell.css

```diff
@@ -42,11 +42,19 @@ text-cell-component code-component .monaco-editor .inputarea.ime-input {
 }
 
 .vs .notebook-preview .rangeHighlight {
-	background-color: rgba(255, 255, 0, 0.2)
+	background-color: yellow;
 }
 
 .vs-dark .notebook-preview .rangeHighlight {
-	background-color: rgba(255, 255, 0, 0.2)
+	background-color: yellow;
+}
+
+.vs .notebook-preview .rangeSpecifcHighlight {
+	background-color: orange;
```

Comment:
We shouldn't be hardcoding color values like this. They should really be defined as a themed color so that users can update them in themes as they wish - for example here's the general editor ones : https://github.com/microsoft/azuredatastudio/blob/main/src/vs/platform/theme/common/colorRegistry.ts#L304

I'm not sure if the md colors are registered like this - but if they aren't then we should still make new entries for this to allow for this functionality.

Reply:
As a refresher, we can leverage notebookStyles for something like this.